tests/integration/test_remoted/test_agent_communication/data/wazuh_multi_agent_status.yaml

```diff
@@ -0,0 +1,13 @@
+- tags:
```

Comment:
`wazuh_multi_agent_protocols_communication.yaml` and this file are the same. You can merge both of them and use the `tags` and `apply_to_modules` labels.